src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,152 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String SHORT_LOGIN = "short";
+    private static final String VALID_LOGIN = "validLogin";
+    private static final String LONGER_LOGIN = "longerLogin";
+    private static final String SHORT_PASSWORD = "short";
+    private static final String VALID_PASSWORD = "validPassword";
+    private static final String LONGER_PASSWORD = "longerPassword";
+    private static final Integer VALID_AGE = 20;
+    private static final Integer UNDERAGE = 17;
+    private static final Integer EXACTLY_AT_AGE = 18;
+    private static final String EXISTING_LOGIN = "existingLogin";
+
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    public void setUp() {
+        StorageDao storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+        storageDao.clear();
```

Comment:
```suggestion
        Storage.people.clear();
```

remove method clear() from StorageDao